src/api/search/src/bin/search.js

```diff
@@ -0,0 +1,80 @@
+const { ELASTIC_MAX_RESULTS_PER_PAGE = 5 } = process.env;
```

Comment:
I don't know why we had this in a `bin` directory, but let's move it to the `src` directly.